resources/docker-build_p2p/keys/create.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEowIBAAKCAQEAy0rUTEY1/HKQozk28NZNG/hb7wAwlBrFWQYgJKZMZf3x7L1c
+xiFHhgTOqSWFeQTI+jON9bBbN93FcS8RBDD9C8djIlCTbi1vhLeNOHwphUp3DCo6
+6PEvzs+8bJDTB0Vc2EdvFhiUSAcGqbayyodywS2YIsc5Nswr6GM6rsrs1tkJ2i8x
+q59J0mg4n6IjootGPc949ThxGshN5yDt+RjrG20QoRET2/N142oU8l3UqQeyVXqP
+nagYZAYmzfC8lAl1LxtpM55JLQwrQPDSO2lVqjerlNYWghpHQv1a13ozDzGoJud8
++kjtZB3z1SGj6YdkeRG7z6nQLaTGBX8xr7yc/QIDAQABAoIBAHh4ZqiwrstkROxF
+OpasGFPr8OcIMMGDv9FKvP+bAQkr3RLTAMh07kKwVJuOHbOir0r+BDFFijjoFlpF
+bwsR0NMLHGc4YrXW/RAkCe8gPSa9AyFVYUWrsA4ZsuqPRE85vpR4eI8aj7zDVSJ7
+nyY/8DQkRdTORMyBX7++FgeBP6rtMS5KGjVlxK+JTMwzyX+VRnYdDkRyfzgxJe8F
+jOktZdTBCIEBUBaDzMMAIH82GZstFgHHInQdJRsD2vWTMnQ8Xc/tDrM8SwrmwDAN
+BL1rNkGMVYZTKWzX9JoqynDIsz8kN1gAsqL0VxsTU+GPD+UH46K7RIoh5H8RaXky
++qQ9YwECgYEA/9E26opcRsMRrpXWssRSH6UZxP/d26IPrSYJucgBz+JbTM/oeXR8
+54lsBqoN1Au4c8PjVgsEX7bu/1JcFP7PeArfOwRze5NF09BmpQnIYWntA6Oi5GRp
+xIcBIjTVX9qZF1wbLtx+4T1TTABgjaHNVa12HtwABVga6+1KyWAyd70CgYEAy3AC
+OKvP1eDz4udStc0WIQfVxNK5DRDjFy3DltZu6Dc5gRiX8t8x0rVSCxiycy6jQqm8
+KbmPLivZIi/SYO/0iENV2cvFj2MrvzhKtPdA3PbjwoWgVCzNOJIh2wYDF4/ataoY
+uyvDghYthHlk8ihF2WK8PW+mPesFcsyg4XhtbkECgYA0ryipKu+QHzJwCynZTTdA
+jyb8fFl7lye1WCUensNIxexZqGPPR59Fq/aUdMc+SV+mb8xM9x3ifHFCIfsW09d0
+eIC6zdLeu1jhI4VQLfCXp1IJstSkRs3FjKw+GDVtdT6cIfrzLrPuvUt/VoCMCquY
+Ho46EsLqVtfMiOUjDe223QKBgAt92kJFQko8mMo6cuueGB/ph2PE23sndwx6I1BD
+8KPhargmIX8wqGnUoDvMO1hSTBqZ5+0bdEERbQSHWwTqyQ9hbloNLq241mHI1j9+
+sAVLVPZKAhJ/LWCCsLLklgwgriQQ5bC8+iBBDHUtPkyHhdqPvz5hLizVN8uST4ws
+91GBAoGBAKIpC4GyPSlCuYH0vQpZdYE+xpMrFTzgNpXL027JEKoswxdTfWRObf9X
++o1y9Zx4ddtAwHtjUOwpT5A6WHXOlYXNcl+3Vt6LWMCYRu1nr7qXgqTOVglkC9d7
+FCwzH3Ja4xRSdKQ3lnflv5ZmKuJmcVn8bhMhA7YhOVM6X6BaMKYF
+-----END RSA PRIVATE KEY-----
```

Comment:
Was it intentional that the key gets uploaded? if not, we might have to add it to the .gitignore-file